### DISCUSSION TITLE
Ability to set fixed star diffculty in multiplayer lobbies

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#15006] [Ideas]
**The problem:**
Player X pics a map, which is 0.5* over the lobby difficulty limit. All players download this map & player Y writes a bit later "This exceeds the star difficulty of this lobby." After that players W and Z will write "But I already downloaded it, so lets play!"

_And thats the whole problem. If the creators/players of the map really know their skill-cap, and one player exceeds it, all players of the lobby will basically "suffer" from it._

**Suggestion:**
When you look into the multiplayer browser, the main theme is, that most of the players write a specific star difficulty (example: "3 - 4.99*") into the lobby title.
It would be very helpful, if there was an (optional) function, to really set a specific limit of a star difficulty for the multiplayer room.

**Comment by Gonzalo-Bruna** [comment]
Commenting for supporting this feature. Sometimes you just want to play in a specific star range and some guy just randomly throws a 1* map or a 20* map and it makes no sense at all to play that. In most cases, you are playing a room picking maps around 5* - 6* of difficulty and a guy who just started playing would pick a 2* map, making everyone else in the lobby not wanting to play the match at all
